.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -38,7 +39,7 @@ const webResult = getDeployTableMessage(core.getInput('WEB', {required: true}));
 const workflowURL = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}`
     + `/actions/runs/${process.env.GITHUB_RUN_ID}`;
 
-let message = `🚀 [Deployed](${workflowURL}) to ${isProd ? 'production' : 'staging'} 🚀`;
+let message = `🚀 [Deployed](${workflowURL}) to ${isProd ? 'production' : 'staging'} in v${version}🚀`;
```

Comment:
```suggestion
let message = `🚀 [Deployed](${workflowURL}) to ${isProd ? 'production' : 'staging'} in version: ${version}🚀`;
```

Nab but I don't love the `v` prefix, I think just `version:` will be more clear.